Core/src/org/sleuthkit/autopsy/guiutils/WrapLayout.java

```diff
@@ -0,0 +1,607 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2019 Basis Technology Corp.
```

Comment:
Year is incorrect.

Also I am not a fan of having multiple classes with identical names is there anyway we can rename one of our two WrapLayout classes?

Reply:
I corrected the year and there is a question in Jira for the second concern.